lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -66,6 +66,17 @@
   .form-actions {
     padding-top: gutter();
   }
+
+  &.-signup {
+    width: 30%;
+    min-width: 256px;
+    margin: 1.5%;
+    display: inline-block;
+
+    .beta-card__row:first-of-type {
+      margin-top: 0;
+    }
+  }
```

Comment:
the only style `.-headerless` applies is to tell the first `.beta-card__row` in `.card` to have 0 margin-top. When you said,

> Maybe we change the base card to add that top margin/padding on the heading and take it off the card

I was trying to point out that the margin is being applied on `.beta_card__row`, and the beta card row is used in multiple places so it won't be easy to remove the margin from this entire class & shift it to the header.